amundsen_application/static/js/ducks/search/sagas.ts

```diff
@@ -8,7 +8,7 @@ import {
 
 import {
   searchExecuteSearch,
-} from '../api/search/v0';
+} from './api';
```

Comment:
The intent behind this folder naming convention was in anticipation of  supporting different versions of apis, and don't want to remove that concept. However I think I have to take this though offline for us to discuss. One thing we haven't discussed is what our search changes will mean for our API versioning. For example does the frontend service need that concept, and if so how does that affect how we should be implementing new features.

Reply:
We can rename to '/search/api/v0.ts`. I think that maintains the versioning as well as keeping the APIs and ducks in the same folder.